proto/proto-backplane-grpc/src/main/java/io/deephaven/proto/util/SharedTicketHelper.java

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.proto.util;
+
+import com.google.protobuf.ByteString;
+import org.apache.commons.codec.binary.Hex;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+public class SharedTicketHelper {
+    public static final char TICKET_PREFIX = 'h';
+    public static final String FLIGHT_DESCRIPTOR_ROUTE = "shared";
+
+    /**
+     * Convenience method to create the flight descriptor path for the given shared variable identifier.
+     *
+     * @param variableId the variable identifier
+     * @return the path
+     */
+    public static List<String> nameToPath(ByteString variableId) {
+        return Arrays.asList(FLIGHT_DESCRIPTOR_ROUTE, Hex.encodeHexString(variableId.toByteArray()));
+    }
+
+    /**
+     * Convenience method to create the flight ticket bytes for the given shared variable identifier.
+     *
+     * @param variableId the variable identifier
+     * @return the ticket bytes
+     */
+    public static byte[] nameToBytes(ByteString variableId) {
+        return (TICKET_PREFIX + "/" + variableId).getBytes(StandardCharsets.UTF_8);
+    }
```

Comment:
I don't _think_ `variableId.toString()` is what we want here. It does:

```java
  @Override
  public final String toString() {
    return String.format(
        Locale.ROOT,
        "<ByteString@%s size=%d contents=\"%s\">",
        toHexString(identityHashCode(this)),
        size(),
        truncateAndEscapeForDisplay());
  }
```

Reply:
Agree that this should not be string-y. Just byte prefix + binary ID.

Reply:
id or sharedId, not name.